mobilitypath_publisher/package.xml

```diff
@@ -18,15 +15,33 @@
 
 <package format="3">
   <name>mobilitypath_publisher</name>
-  <version>3.3.0</version>
-  <description>This published MobilityPath Messages.</description>
-  <maintainer email="CARMA@dot.gov">carma</maintainer>
-  <license>Apache License 2.0</license>
-  <buildtool_depend>catkin</buildtool_depend>
-  <depend>roscpp</depend>
+  <version>1.0.0</version>
```

Comment:
lets make the version 4.0 so it remains higher than the old value and indicates the substantial change.